src/main/kotlin/io/github/mojira/risa/infrastructure/Jira.kt

```diff
@@ -51,7 +53,7 @@ fun getTicketsForSnapshot(jiraClient: JiraClient, currentSnapshot: Snapshot): Li
 
 @SuppressWarnings("MaxLineLength")
 private fun getJql(currentSnapshot: Snapshot): String =
-    "project = MC AND affectedVersion = \"${currentSnapshot.name}\" AND (status = Open OR status = Reopened OR resolution in (\"Works As Intended\", \"Fixed\", \"Awaiting Response\", \"Unresolved\", \"Won't Fix\"))"
+    "project = MC AND affectedVersion = \"${currentSnapshot.name}\" AND created > ${currentSnapshot.releasedDate.minus(1L, ChronoUnit.DAYS).toEpochMilli()} AND (status = Open OR status = Reopened OR resolution in (\"Works As Intended\", \"Fixed\", \"Awaiting Response\", \"Unresolved\", \"Won't Fix\"))"
```

Comment:
You might need to convert it to Jira format

Reply:
I believe Jira support pure numbers as dates